infra/prod/_modules/app_services/app_service_fims_oidc_provider.tf

```diff
@@ -0,0 +1,34 @@
+module "appservice_fims" {
+  source = "github.com/pagopa/terraform-azurerm-v3//app_service?ref=v7.67.1"
+
+  plan_type = "internal"
+  plan_name = "${var.product}-oidc-provider-plan"
+  sku_name  = "B3"
+
+  name                = "${var.product}-oidc-provider-app"
+  resource_group_name = var.resource_group_name
+  location            = var.location
+
+  always_on        = true
+  node_version     = "18-lts"
+  app_command_line = "npm run start"
```

Comment:
we should avoid using `npm` to start our node projects, the startup command should be `node .` (is it the default one?)

Reply:
`npm run start` is set by our app service TF module, and the current FIMS app service uses it.
Do you think the module shall be updated too?

Ok for this app service, I'll update it

Reply:
I think the module should be updated, but I don't know if this change will have impact on production.

Using `npm start` is a bad practice, but maybe our legacy projects rely on it.